### ISSUE TITLE
Avira free anti virus version 14.0.7.468: NVDA does not read the items in the tree view opened through Extras submenu > Detections List > Help button

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4805]
**Reported by bhavyashah on 2015-01-14 13:17**
Steps to reproduce
1 Download, install and open Avira free anti virus version 14.0.7.468.
2 Go to Extras submenu > Detections list > Help button.
3 You land on a tree view, where:
Expected Result : NVDAD should read the names of the tree view items as it usually does, at least through alternate methods like object navigation and screen review.
Actual Result : NVDA does not announce the name of the tree view item, instead simply says ‘tree view item x of y level z’, where x, y and z are variables of the item/total items/level number. Object navigation and screen review were unable to reveal any information either.
Narrator remained totally silent, didn’t even announce the entering of a tree view, or that the user was actually navigating tree view items x of y level z. Narrator's silence may possibly be due to my inexperience in using it.
Test Environment : Windows 7 Ultimate 32 bit
Avira free anti virus version 14.0.7.468
NVDA version 2014.4 stable

**Comment by leonardder** [comment]
Closing as duplicate of #4804, which is most likely the same underlying issue.
